airbyte-config/config-persistence/src/main/java/io/airbyte/config/persistence/split_secrets/SecretPersistence.java

```diff
@@ -34,6 +34,9 @@ static Optional<SecretPersistence> getLongLived(final @Nullable DSLContext dslCo
       case VAULT -> {
         return Optional.of(new VaultSecretPersistence(configs.getVaultAddress(), configs.getVaultPrefix(), configs.getVaultToken()));
       }
+      case AWS_SECRET_MANAGER -> {
+        return Optional.of(new AWSSecretManagerPersistence(configs.getAwsAccessKey(), configs.getAwsSecretAccessKey()));
+      }
```

Comment:
any reason to not support IAM role based authentication ?

Reply:
The Airbyte project at large doesn't seem to support role based auth at all, which is pretty rough.